elixir/apps/api/test/api/controllers/gateway_controller_test.exs

```diff
@@ -0,0 +1,174 @@
+defmodule API.GatewayControllerTest do
+  use API.ConnCase, async: true
+  alias Domain.Gateways.Gateway
+
+  setup do
+    account = Fixtures.Accounts.create_account()
+    actor = Fixtures.Actors.create_actor(type: :api_client, account: account)
+    gateway_group = Fixtures.Gateways.create_group(%{account: account})
+
+    %{
+      account: account,
+      actor: actor,
+      gateway_group: gateway_group
+    }
+  end
+
+  describe "index/2" do
```

Comment:
There are no tests for `create/2`.

Reply:
Yeah, there's no `create` function for the `gateway_controller`, because you shouldn't be able to create a gateway through the API right?

Reply:
@bmanifold ah, I was expecting there we would have a way to create a token, but now realized it's for the group. Sorry.